sdk/storage/storage-file-datalake/src/generated/src/storageClientContext.ts

```diff
@@ -16,6 +16,7 @@ export class StorageClientContext extends coreHttp.ServiceClient {
   url: string;
   version: string;
   resource: string;
+  xMsLeaseDuration?: number;
```

Comment:
seems weird.

Reply:
The datalake SDK doesn't have methods for leasing operation, so no interface would need to set this value.
.Net SDK also doesn't have leasing operation methods.

The change is because the Swagger spec added an option to put the parameter in client instance instead of method: : "x-ms-parameter-location": "client". The SDK doesn't expose any interface, so it has no impact on customer. But I'd still need to check with Sean why made this change.